packages/amplify_storage_s3/ios/Classes/FlutterValidationException.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Foundation
+import Amplify
+import AmplifyPlugins
+
+class AmplifyFlutterValidationException: AmplifyError {
```

Comment:
Do you still need this since there is one of these in amplify_core as well?

Reply:
We only have it in Core for Android.  I was having compilation issues in XCode, and decided to move on in the interest of time.